docs/source/nbconvert_library.ipynb

```diff
@@ -511,27 +511,18 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The following demonstration, as requested in [a GitHub issue](https://github.com/ipython/nbconvert/pull/137#issuecomment-18658235), adds the ability to exclude a cell by index. \n",
+    "The following demonstration, as requested in [a GitHub issue](https://github.com/jupyter/nbconvert/pull/137#issuecomment-18658235), adds the ability to exclude a cell by index. \n",
```

Comment:
To be honest, I have no idea... I didn't handle that repo split/merge period myself...

Not losing this kind of historical context is why I recently created [the Jupyter attic](https://github.com/jupyter-attic), so we can move entire repos (along with issues, wikis, etc) out of the way without killing our history.

@minrk might remember what happened there?